### ISSUE TITLE
The latest version of amdgpu-pro is not compatible

### ISSUE THREAD

**Issue by M4gma** [Mystro256/amdgpu-pro-rpm#1]
error: File ~/rpmbuild/SOURCES/amdgpu-pro_16.30.3-306809.tar.xz: file or directory not found

It happens because the last driver release is 16.30.3-315407 (file amdgpu-pro_16.30.3-315407.tar.xz)

**Comment by SevM** [comment]
Edit the spec file and change the version number to 315407 that seemed to help me build the rpm's but now I'm not able to boot into graphical user.target. Any tips on this would be greatly appreciated. Seems to not be able to boot gdm on fedora 24

**Comment by Mystro256** [comment]
Indeed, I can't seem to get it to work. I think we may need to wait for generic or RedHat binaries.

**Comment by SevM** [comment]
I just came across this. https://github.com/Lucretia/vulkan-overlay/issues/12 Possible fix if the patch noted there is used. Don't have the time to test out tonight but possibly this weekend. Yea if only we could get a full driver deployment for all the Major Distros. Amd might see some traction on their new driver.

**Comment by Mystro256** [comment]
Hmm, I think maybe they're waiting until the driver is out of beta. I spoke to a couple amd devs and they said that a RHEL build is coming, which should be compatable with fedora.

At the moment I'm using the staging kernel and dev builds of mesa, llvm and the xorg driver:

http://copr-fe.cloud.fedoraproject.org/coprs/mystro256/polaris-gfx/

**Comment by leifliddy** [comment]
I'm using the mystro256 copr repo (RX 480 Nitro+ 8GB) , but something is off. When I run the Unigine Heaven Benchmark, it freezes within 2 minutes every time. I've changed to the stock Fedora 4.7.2-201 kernel, but have left the xorg-x11-drv-amdgpu and mesa 12.1.x installed from the mystro repo. The Unigine Heaven now runs without crashing. The only differences between the two kernels is that mystro256 kernel-4.7.2-902.amd has these options enabled:
CONFIG_DRM_AMDGPU_CIK=y
CONFIG_DRM_AMD_DAL=y
CONFIG_DRM_AMD_DAL_VBIOS_PRESENT=y
CONFIG_DRM_AMD_DAL_DCE8_0=y
CONFIG_DRM_AMD_DAL_DCE10_0=y
CONFIG_DRM_AMD_DAL_DCE11_0=y
CONFIG_DRM_AMD_DAL_DCE11_2=y

**Comment by Mystro256** [comment]
Can you post any issues you have with the staging kernel here:
https://www.phoronix.com/forums/forum/linux-graphics-x-org-drivers/open-source-amd-linux/889794-amd-staging-4-6-for-fedora-24-amdgpu

This github repo is just for the amdgpu-pro driver

**Comment by Mystro256** [comment]
I tried repacking the RHEL 7.2 drivers, but no luck so far. FOSS drivers work pretty well and with radv, so I may abandon this if I can't figure out a way to get this to work.
